packages/expo-modules-core/ios/Core/Modules/CoreModuleHelper.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface CoreModuleHelper : NSObject
+
++ (NSString *)getVersion;
+
+@end
```

Comment:
```suggestion
@interface CoreModuleHelper : NSObject

NS_ASSUME_NONNULL_BEGIN

+ (NSString *)getVersion;

@end

NS_ASSUME_NONNULL_END
```
You can make all types nonnullable by default. FYI, the same can be achieved with the `_Nullable` keyword added to the argument/return type.